drivers/google/compute_util.go

```diff
@@ -277,6 +275,17 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 	return nil
 }
 
+// tags computes the tags for the instance.
+func tags(d *Driver) []string {
```

Comment:
@nathanleclaire ok for the better name. Making the method a function on the driver gives too much importance to it I think. I'd rather leave it as is.